### PR TITLE
Cosmetic changes to README, add .gitignore

### PR DESCRIPTION
**Pull request by akien-mga**
Fixed some typos in tinyexr.h.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -408,7 +408,7 @@ See `example/deepview` for actual usage.
 
 ## Defines
 
-* TINYEXR_USE_MINIZ  Set `1`(default) to use embedded miniz compression. If you want zlib or custom zlib compatible library, set `0` and link zlib library.
+* TINYEXR_USE_MINIZ  Set `1` (default) to use embedded miniz compression. If you want zlib or custom zlib compatible library, set `0` and link zlib library.
```
Comment (akien-mga) [inline README.md:411]: BTW, it seems like this section is outdated documentation? It doesn't match `### Compile flags` earlier and seems redundant. The `## TODO` section also seems to overlap with the earlier `Current status` section.

Comment (syoyo) [reply README.md:411]: Good catch. Yes we should remove this `## Defines` section.

Comment (akien-mga) [reply README.md]: Done.

Comment (syoyo) [reply README.md]: :pray:

**Comment by syoyo** [comment]
Super awesome! Merged!

**Pull request merged**